x/perp/v2/keeper/admin.go

```diff
@@ -174,3 +174,33 @@ func (k admin) UnsafeChangeCollateralDenom(
 	k.Collateral.Set(ctx, denom)
 	return nil
 }
+
+// ShiftPegMultiplier: Edit the peg multiplier of an amm pool after making sure
+// there's enough money in the perp fund to pay for the repeg. These funds get
+// send to the vault to pay for trader's new net margin.
+func (k admin) ShiftPegMultiplier(
+	ctx sdk.Context,
+	pair asset.Pair,
+	newPriceMultiplier sdk.Dec,
+	sender sdk.AccAddress,
+) error {
+	if err := k.SudoKeeper.CheckPermissions(sender, ctx); err != nil {
+		return err
+	}
+	return k.UnsafeShiftPegMultiplier(ctx, pair, newPriceMultiplier)
```

Comment:
I would do this function private as calling it from outside can make the application inconsistent. If we can avoid it that is good.

Reply:
Done. As an even better step, I gave the sudo root address a convenience getter function inside `testapp.go` and deleted "Unsafe..." entirely for both messages

fe1fe03cb9cf99369486d7455c06a3d43c5eef25